packages/models/src/projects/project_field.rs

```diff
@@ -29,19 +29,19 @@ pub enum ProjectArea {
     Politics = 10,
 }
 
-impl ProjectArea {
-    pub fn all() -> Vec<ProjectArea> {
+impl ProjectField {
+    pub fn all() -> Vec<ProjectField> {
```

Comment:
ProjectField::VARIANTS is same with this.
Translate derive makes VARIANTS static variables in ProjectField.

And also it makes `pub fn variants(lang, &Language) -> Vec<String>`